app/components/Settings/OptionsDropdown/SQLOptions.react.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {flatten} from 'ramda';
+import Select from 'react-select';
+
+export default class SQLOptions extends Component {
+
+    static propTypes = {
+        selectedTable: PropTypes.any,
```

Comment:
If we can't use `PropTypes.string` here, then we should bring back the comment `// See type of react-select's Select 'value' property`.